sklearn/linear_model/_passive_aggressive.py

```diff
@@ -24,7 +24,7 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
     max_iter : int, default=1000
         The maximum number of passes over the training data (aka epochs).
         It only impacts the behavior in the ``fit`` method, and not the
-        :meth:`PassiveAggressive.partial_fit` method.
+        `partial_fit` method.
```

Comment:
can't we link to the method?

Reply:
I could not make it work for the regressor. But now I see that there is no methods generated for the API page that explain why sphinx don't find references: https://scikit-learn.org/dev/modules/generated/sklearn.linear_model.PassiveAggressiveRegressor.html

I will check more what is the reason.

Reply:
OK it was listed as a function in `classes.rst` and we don't generate documentation for methods then.